pkg/build/nodeimage/internal/kube/builder_remote.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
nit: Should probably use the current year for new files?

Reply:
Fixed @stmcginnis thanks! please re-lgtm if it looks good.